### DISCUSSION TITLE
Delete account endpoint force to enter the password again

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#4715] [Feature Requests]
The SDK is the permissions allows it, the user can delete its own account as long it is signed in. Thinking about forcing to pass the password to confirm that the user is the owner of the account. I know we will have a problem as the SDK allows to delete other users account, perhaps it can be moved to the `me`.

During my current project, I re-authenticate the user to make sure the user is correct, but a techie guy could learn that only needs to call the endpoint passing the user id.

Just an idea :)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
